src/views/projects/Patches.svelte

```diff
@@ -118,47 +107,62 @@
 
 <div class="patches">
   <div style="margin-bottom: 1rem;">
-    <TabBar {options} on:select={switchState} active={state} />
+    <div style="display: flex; gap: 0.5rem;">
+      {#each options as option}
+        {#if !option.disabled}
```

Comment:
You can switch the body of the if statement around and make the check `#if option.disabled`. This is easier to read.

Reply:
f09d68094e03b82194194eb87f8c9d67509c0431